src/domainModules/collectionMammals/components/MammalForm/transformations/input/localityInformation.js

```diff
@@ -11,7 +11,9 @@ export default function transformLocalityInformation(localityInformation) {
     return localityInformation
   }
 
-  const currentCuratedLocalityMap = localityInformation.curatedLocalities.reduce(
+  const rawCuratedLocalities = localityInformation.curatedLocalities || []
+
+  const currentCuratedLocalityMap = rawCuratedLocalities.reduce(
```

Comment:
I kind of like this short way of ensuring an array
`const currentCuratedLocalityMap =  (localityInformation.curatedLocalities || []).reduce(`